cli/command/image/build.go

```diff
@@ -182,6 +182,10 @@ func runBuild(dockerCli command.Cli, options buildOptions) error {
 		remote        string
 	)
 
+	if options.compress && options.stream {
+		fmt.Fprintf(dockerCli.Err(), "--compress has no effect when used with --stream")
```

Comment:
nitnit: or `"--compress ignored when used with --stream"`?

Reply:
Right; so in that case it's even better to error out, because otherwise the option is ignored now (so people can keep using it, it has no effect), but when we implement compression for streaming, the behavior changes, and now the flag _does_ have effect.

Producing an error now, makes sure that nobody uses it

Reply:
alright, seems fair then :angel: